webpack/components/extensions/Hosts/BulkActions/BulkActionsConstants.js

```diff
@@ -0,0 +1,7 @@
+const BULK_ACTIONS_OSFAMILY = {
```

Comment:
I found myself confused as I was reading "OSFamily" throughout this PR. These are not OS families.

```suggestion
const PACKAGE_CONTENT_TYPE_NAMES = {
```

Reply:
I wonder if we should rename the whole file or leave it with a generic name for future extension.